e2e_test/batch/order/test_order_requirers_index.part

```diff
@@ -0,0 +1,41 @@
+# description: Test lookup joins with order
+
+statement ok
+SET RW_IMPLICIT_FLUSH TO true;
+
+statement ok
+create table t1 (a int, b int, c int);
+
+statement ok
+INSERT INTO t1 VALUES (10, 0, 35), (10, -2, 28), (-2, 1, 81), (6, -5, 29), (-2, -5, 51), (8, 2, 17), (1, 4, 54), (10, 5, 53), (-1, 3, 96), (10, -1, 70), (5, 2, 50), (-10, 3, 55), (9, -2, 32), (-8, 1, 91), (-2, -1, 75), (9, 4, 13);
+
+statement ok
+create index idx1 on t1(a, b, c);
+
+# Index on all required columns
+query I rowsort
+select * from t1 order by a;
+----
+-1 3 96
+-10 3 55
+-2 -1 75
+-2 -5 51
+-2 1 81
+-8 1 91
+1 4 54
+10 -1 70
+10 -2 28
+10 0 35
+10 5 53
+5 2 50
+6 -5 29
+8 2 17
+9 -2 32
+9 4 13
```

Comment:
Apparently once we use bigger test cases, there is an issue with the order here

Reply:
Should we create an issue for this?